unittests/finalizer_update_tests.cpp

```diff
@@ -120,42 +120,42 @@ BOOST_AUTO_TEST_CASE(savanna_set_finalizer_multiple_test) { try {
    auto pubkeys5 = fin_keys.set_finalizer_policy(5u).pubkeys;
    b = t.produce_block(); // proposed: pubkeys3, pubkeys4, pubkeys5
    verify_block_finality_policy_diff(b, 5, pubkeys5.back());
+   t.check_head_finalizer_policy(2u, pubkeys2); // 3 blocks after pubkeys3 (b5 - b0), pubkeys2 should still be active
    t.produce_block();  // pending: pubkeys3, proposed: pubkeys4, pubkeys5
    auto pubkeys6 = fin_keys.set_finalizer_policy(6u).pubkeys;
    b = t.produce_block(); // pending: pubkeys3, proposed: pubkeys4, pubkeys5, pubkeysr6
    verify_block_finality_policy_diff(b, 6, pubkeys6.back());
    auto pubkeys7 = fin_keys.set_finalizer_policy(7u).pubkeys;
-   t.check_head_finalizer_policy(2u, pubkeys2); // 5 blocks after pubkeys3 (b5 - b0), pubkeys2 should still be active
+   t.check_head_finalizer_policy(3u, pubkeys3); // 4 blocks after pubkeys3 (b6 - b0), pubkeys3 should be active
    b = t.produce_block(); // active: pubkeys3, pending: pubkeys5, proposed: pubkeysr6, pubkeys7 -- pubkeys4 was garbage collected
    verify_block_finality_policy_diff(b, 7, pubkeys7.back());
    auto pubkeys8 = fin_keys.set_finalizer_policy(8u).pubkeys;
-   t.check_head_finalizer_policy(3u, pubkeys3); // 6 blocks after pubkeys3 (b6 - b0), pubkeys3 should be active
    b = t.produce_block(); // pending: pubkeys5, proposed: pubkeysr6, pubkeys7, pubkeys8
    verify_block_finality_policy_diff(b, 8, pubkeys8.back());
    auto pubkeys9 = fin_keys.set_finalizer_policy(9u).pubkeys;
-   t.check_head_finalizer_policy(3u, pubkeys3);
+   t.check_head_finalizer_policy(4u, pubkeys4);
    b = t.produce_block(); // pending: pubkeys5, proposed: pubkeysr6, pubkeys7, pubkeys8, pubkeys9
    verify_block_finality_policy_diff(b, 9, pubkeys9.back());
    auto pubkeys10 = fin_keys.set_finalizer_policy(10u).pubkeys;
-   t.check_head_finalizer_policy(3u, pubkeys3); // 7 blocks after pubkeys4, pubkeys4 should still be active
+   t.check_head_finalizer_policy(5u, pubkeys5); // 7 blocks after pubkeys4, pubkeys4 should still be active
```

Comment:
comment is clearly incorrect.

Reply:
To be fixed by https://github.com/AntelopeIO/spring/issues/398